WooCommerce/Classes/ServiceLocator/ProductImageUploader.swift

```diff
@@ -243,12 +254,30 @@ final class ProductImageUploader: ProductImageUploaderProtocol {
     func reset() {
         statusUpdatesExcludedProductKeys = []
         statusUpdatesSubscriptions = []
+        backgroundSubscriptions = []
         imageUploadSubscriptions = []
         activeUploadsPublisher = []
 
         actionHandlersByProduct = [:]
         imagesSaverByProduct = [:]
     }
+
+    private func scheduleUploadInProgressNotification() {
+        guard !activeUploadsPublisher.isEmpty else { return }
+
+        let notification = LocalNotification(scenario: .productImageBackgroundUpload)
+        let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 1, repeats: false)
+        Task {
+            await LocalNotificationScheduler(pushNotesManager: ServiceLocator.pushNotesManager).schedule(notification: notification,
+                                                                                                         trigger: trigger, remoteFeatureFlag: nil)
+        }
+    }
+
+    @objc private func appDidEnterBackground() {
+        if !activeUploadsPublisher.isEmpty {
+            self.scheduleUploadInProgressNotification()
+        }
```

Comment:
Nit: since you're checking `activeUploadsPublisher` in `scheduleUploadInProgressNotification`, this extra check seems redundant. I'd suggest removing the check and renaming the method to `scheduleUploadInProgressNotificationIfNeeded`